Subject: [PATCH] fix file_contexts.subs_dist for poky

This file is used for Linux distros to define specific pathes 
mapping to the pathes in file_contexts.

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 config/file_contexts.subs_dist |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/config/file_contexts.subs_dist b/config/file_contexts.subs_dist
index 32b87a4..ebba73d 100644
--- a/config/file_contexts.subs_dist
+++ b/config/file_contexts.subs_dist
@@ -5,3 +5,11 @@
 /usr/lib32 /usr/lib
 /usr/lib64 /usr/lib
 /var/run/lock /var/lock
+/etc/init.d /etc/rc.d/init.d
+/var/volatile/log /var/log
+/var/volatile/run /var/run
+/var/volatile/cache /var/cache
+/var/volatile/tmp /var/tmp
+/var/volatile/lock /var/lock
+/var/volatile/run/lock /var/lock
+/www /var/www
-- 
1.7.5.4

